src/main.java

```diff
@@ -1,6 +1,6 @@
-public class java {
+public class main {
+
     public static void main(String[] args) {
         System.out.println("Hello, World!");
```

Comment:
create a new class with a method that prints "Hello, World!", and use it here instead of this command